tardis/plasma/properties/nlte.py

```diff
@@ -0,0 +1,17 @@
+from tardis.plasma.properties.base import BasePlasmaProperty
```

Comment:
@wkerzendorf Think I've found a better way of doing this now, without having to output and read in values from previous iterations from model.py. Created this new property class to store values from previous iterations. The values are set just prior to all of the properties being updated. I think it's a lot more straightforward!